src/components/Header/_Header.scss

```diff
@@ -1,96 +1,99 @@
-.navbar, .logo-pic img {
-  width: 60px;
+.navbar,
+.logo-pic img {
+    width: 60px;
 }
 
 .logo-pic img {
-  margin-top: 10px;
+    margin-top: 10px;
 }
 
 .nav {
-  position: sticky;
-  z-index: 2;
-  width: 100%;
-  height: 65px;
-  padding: 0 20px;
-  display: flex;
-  justify-content: space-between;
+    position: sticky;
+    z-index: 2;
+    width: 100%;
+    height: 65px;
+    padding: 0 20px;
+    display: flex;
+    justify-content: space-between;
+    background-color: #faf9f9;
```

Comment:
Sorry about all the changes, my prettier config has automatically edited a bunch of stuff making it impossible to see the minor changes I made. I'll have to find a solution for that.

@koral14 I thought I had added a light grey colour to the navbar.